request-postgres/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:12-slim
+
```

Comment:
For security, consider running as a user
```suggestion
USER 1001
```

Reply:
Not sure what this would achieve?

Reply:
https://docs.docker.com/develop/develop-images/instructions/#user
> If a service can run without privileges, use USER to change to a non-root user.

It's a security best practice. This same comment applies to all the docker images.

Reply:
Decided to leave "as is" due to all other docker images